EIPS/eip-5656.md

```diff
@@ -133,34 +135,40 @@ post:
 0101010101010101010101010101010101010101010101010101010101010101
 ```
 
+gas used: 6
+
 `MCOPY 0 1 8` - copy 8 bytes from offset 1 to offset 0 (overlapping).
 
 pre:
 
 ```
-000102030405060708 000000000000000000000000000000000000000000000000
+000102030405060708 0000000000000000000000000000000000000000000000
 ```
 
 post:
 
 ```
-010203040506070808 000000000000000000000000000000000000000000000000
+010203040506070808 0000000000000000000000000000000000000000000000
 ```
 
+gas used: 6
+
 `MCOPY 1 0 8` - copy 8 bytes from offset 0 to offset 1 (overlapping).
 
 pre:
 
 ```
-000102030405060708 000000000000000000000000000000000000000000000000
+000102030405060708 0000000000000000000000000000000000000000000000
 ```
 
 post:
 
 ```
-000001020304050607 000000000000000000000000000000000000000000000000
+00000102030405060708 00000000000000000000000000000000000000000000
```

Comment:
This does not look correct to me. You copy 8 bytes from offset 0 to offset 1. So, one copies:

`0001020304050607` (so 8 bytes from offset 0). However the output now also includes the moved `08` byte?

Reply:
yea, that's another typo. i'll update and remove that extra leading `00` since it only brings confusion -- thank you!